autotest/ogr/ogr_shape.py

```diff
@@ -4818,6 +4818,37 @@ def test_ogr_shape_ldid_and_cpg():
     gdal.Unlink('/vsimem/tmp.dbf')
     gdal.Unlink('/vsimem/tmp.cpg')
 
+###############################################################################
+# Test ExecuteSQL() arithmetic expression
+
+
+def test_ogr_shape_arithmetic_expression():
```

Comment:
this test should rather be in ogr_sql_test.py

Reply:
Done